XamlControlsGallery/XamlControlsGallery.csproj

```diff
@@ -11,7 +11,7 @@
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
     <TargetPlatformVersion>10.0.18362.0</TargetPlatformVersion>
-    <TargetPlatformMinVersion>10.0.17134.0</TargetPlatformMinVersion>
+    <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
```

Comment:
I understand needing to update the NuGet package. I don't understand why the minimum platform version is changed.

Reply:
About this and your other edit about the ink toolbar page, honestly not sure how these got added in - i may have pulled from an older version of the .csproj file. I'll fix these.

Reply:
Just to make sure before i change it back - @ranjeshj does the newest pre-release package require a higher/newer min platform version?

Reply:
Nope. No changes required for the platform versions.

Reply:
Thanks, changing back in upcoming commit.